frontend/packages/dev-console/src/components/import/section/ResourceSection.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { connectToFlags, FlagsObject } from '@console/internal/reducers/features';
+import {
+  FLAG_KNATIVE_SERVING_SERVICE,
+  ServiceModel,
+  KnativeServingModel,
+} from '@console/knative-plugin';
+import { getBadgeFromType } from '@console/shared';
+import { useAccessReview } from '@console/internal/components/utils';
+import { getActiveNamespace } from '@console/internal/actions/ui';
+import { RadioButtonField } from '../../formik-fields';
+import FormSection from './FormSection';
+import './ResourceSection.scss';
+
+type ServerlessSectionProps = {
+  flags: FlagsObject;
+};
+
+const ResourceSection: React.FC<ServerlessSectionProps> = ({ flags }) => {
+  const radioOptions = [
+    {
+      label: (
+        <div>
+          Kubernetes
```

Comment:
I'm not sure if this is the right label since we produce a Route either way, which is an OpenShift resource. @serenamarie125

Reply:
It is not - there was an update of designs after this PR was put up. I think I covered that in my review. I believe it's called `Knative Service` now.

Reply:
Thanks, missed that